cmd/api/main.go

```diff
@@ -187,28 +179,39 @@ func main() {
 	ctx, cancel := context.WithCancel(ctx)
 	defer cancel()
 	mux := runtime.NewServeMux()
-	err = v1alpha2pb.RegisterResultsHandlerFromEndpoint(ctx, mux, ":"+serverConfig.GRPC_PORT, opts)
+	err = v1alpha2pb.RegisterResultsHandlerFromEndpoint(ctx, mux, ":"+serverConfig.SERVER_PORT, opts)
```

Comment:
nit - I would make it clearer that `mux` is the http/REST handler by renaming it to `httpMux`

Reply:
Agree, renamed.